docs/tutorial/part-four/index.md

```diff
@@ -78,11 +78,12 @@ Run this command in a new terminal window:
 
 ```shell
 gatsby new tutorial-part-four https://github.com/gatsbyjs/gatsby-starter-hello-world
+cd tutorial-part-four
```

Comment:
I think this command should be described the first time it's used. Maybe delete it from here?